flink-runtime/src/test/java/org/apache/flink/runtime/checkpoint/channel/ChannelStateCheckpointWriterTest.java

```diff
@@ -121,13 +122,15 @@ public void testSmallFilesNotWritten() throws Exception {
         writer.writeInput(new InputChannelInfo(1, 2), buffer);
         writer.completeOutput();
         writer.completeInput();
-        assertTrue(result.isDone());
-        assertEquals(0, checkpointsDir.list().length);
-        assertEquals(0, sharedStateDir.list().length);
+        assertThat(result.isDone()).isTrue();
+        String[] list = checkpointsDir.list();
+        int length = list.length;
+        assertThat(length).isEqualTo(0);
+        assertThat(sharedStateDir.list().length).isEqualTo(0);
```

Comment:
```suggestion
        assertThat(checkpointsDir).isEmptyDirectory();
        assertThat(sharedStateDir).isEmptyDirectory();
```
Could be a bit simplified